docs/cow-protocol/concepts/how-it-works/how-intents-are-formed.mdx

```
@@ -0,0 +1,318 @@
+beforeAllFees.sellAmount = quote.sellAmount + quote.feeAmount
+
+Network Costs (buy token) = (quote.buyAmount × quote.feeAmount) / quote.sellAmount
+
+beforeAllFees.buyAmount = quote.buyAmount + protocolFeeAmount + Network Costs (buy token)
+
+```
+
+
+### Buy order
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)